apmpackage/apm/0.1.0/agent/input/template.yml.hbs

```diff
@@ -2,3 +2,6 @@ apm-server:
     host: {{host}}
     secret_token: {{secret_token}}
     rum.enabled: {{enable_rum}}
+    kibana:
+        enabled: true
```

Comment:
```suggestion
```

`enabled: true` is implied, it's not necessary to explicitly specify it.

Reply:
Sure, I just liked to be explicit.